### ISSUE TITLE
Build error updating to 0.7.0

### ISSUE THREAD

**Issue by lsg-richard** [willviles/ember-useragent#20]
When updating from 0.6.1 to 0.7.0 I'm seeing the following build error:

```
while import node_modules/ua-parser-js/src/ua-parser.js: found an unknown transformation name cjs. Available transformNames are: amd
```

Is there a missing dependency perhaps?

**Comment by willviles** [comment]
Hmm, I can't seem to reproduce this error @lsg-richard. Which version of Ember are you running?

Try locking to this commit: https://github.com/willviles/ember-useragent/commit/83cfe0ca3ae5be992e196e8a29355bb07c25483b. I've added [ember-cli-cjs-transform](https://github.com/rwjblue/ember-cli-cjs-transform) to the `ember-addon` `after` prop.

**Comment by lsg-richard** [comment]
I'm on Ember 3.3.1 and Ember CLI 3.3.0.

Tried locking to that commit and no difference.

Maybe we should ask on the ember-cli--cjs-transform project?

**Comment by raido** [comment]
I ran into same issue, Ember-CLI 3.3.0.

Perhaps "ember-auto-import" would be useful here instead of "ember-cli-cjs-transform"?

**Comment by buschtoens** [comment]
We _probably_ need to install in the host app. Didn't think that one through, sorry folks.

If somebody wants to take this before I get to it: 

- https://ember-cli.com/api/classes/Blueprint.html#method_addAddonToProject

**Comment by willviles** [comment]
Fix released in `0.7.1`. Thanks for the help @lsg-richard, @raido, @buschtoens.
